src/timer.js

```diff
@@ -251,11 +251,11 @@ class Timer {
     const time = this.time();
     return template
       .replace('%label', this._label ? `${this._label}: ` : '')
-      .replace('%d', time.d)
-      .replace('%h', time.h)
-      .replace('%m', time.m)
+      .replace('%ms', time.ms)
       .replace('%s', time.s)
-      .replace('%ms', time.ms);
+      .replace('%m', time.m)
+      .replace('%h', time.h)
+      .replace('%d', time.d);
   }
```

Comment:
good catch @antcoop thanks for opening the PR!

- can you please also change order for consistency, here https://github.com/eyas-ranjous/timer-node/blob/master/src/timer.js#L190-L195

- and if possible to add an assertion that covers this case, here https://github.com/eyas-ranjous/timer-node/blob/master/test/timer.test.js#L177-L184 
```js
expect(timer.format('%s s %ms ms')).to.equal(`${time.s} s ${time.ms} ms`);
```

cheers!